core/src/main/scala/org/apache/spark/deploy/mesos/MesosDriverDescription.scala

```diff
@@ -40,24 +41,28 @@ private[spark] class MesosDriverDescription(
     val cores: Double,
     val supervise: Boolean,
     val command: Command,
-    val schedulerProperties: Map[String, String],
+    schedulerProperties: Map[String, String],
```

Comment:
Removing `val` means `schedulerProperties` is not saved as a field of the class.  It's just used during construction.  I could also make this `private val` if that's more standard in this project.